CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 
-Apollo project is [Apache licensed](LICENSE.md) and accepts
+Sourced ML project is [Apache licensed](LICENSE.md) and accepts
```

Comment:
I also noticed "go-git" at line 35, would you kindly

Reply:
done